### PR TITLE
Alex dev

### PR DESCRIPTION
**Pull request by LorenzoMerotto**
Updated CDSeq/harmony dependencies.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -58,6 +59,7 @@ Remotes:
     omnideconv/BayesPrism,
     omnideconv/MOMF,
     omnideconv/SCDC,
+    omnideconv/harmony,
```
Comment (grst) [inline DESCRIPTION:62]: this should absolutely not be necessary :shrug:

File: vignettes/omnideconv_vignette.Rmd [hunk 2]
```diff
@@ -20,8 +20,12 @@ knitr::opts_chunk$set(
   message = FALSE,
   warning = FALSE
 )
+
+
+library(omnideconv)
```
Comment (grst) [inline vignettes/omnideconv_vignette.Rmd:25]: this shouldn't be necessary either

**Comment by grst** [comment]
Ok, so now there seems a problem with scaden, but CDSeq works

**Comment by LorenzoMerotto** [comment]
Yes. 
Somehow in the vignette Scaden is supposed to be installed alongside autogenes but the package is not found. 
I never had this issue before though

**Pull request merged**